src/uwtools/resources/rocoto.jsonschema

```diff
@@ -0,0 +1,59 @@
+{
+    "workflow": {
+        "attrs": {
+            "type": "object",
+            "properties": {
+                "realtime": {
+                    "type": "string",
+                    "enum": [
+                        "true",
+                        "false",
+                        "T",
+                        "F",
+                        "t",
+                        "f",
+                        "True",
+                        "False",
+                        "FALSE",
+                        "TRUE"
+                    ]
+                }
+                "scheduler": {
+                    "type": "string",
+                    "enum": [
+                        "sge",
+                        "lsf",
+                        "lsfcray",
+                        "ll",
+                        "torque",
+                        "moabtorque",
+                        "moab",
+                        "pbspro",
+                        "slurm",
+                        "cobalt"
+                    ]
+                }
+                "cycylethrottle": {
+                    "type": "integer",
+                    "minimum": 0
+                }
+                "taskthrottle": {
+                    "type": "integer",
+                    "minimum": 0
+                }
+            },
+            "required": [ "realtime", "scheduler"]
+        }
+        "cycledefs": {
+            "type": "string",
+            "pattern": "^\[\d{10},\d{10},(0[0-9]|1[0-9]|2[0-4]):([0-5][0-9]):([0-5][0-9])\]$"
+        }
+        "log": {
+            "type": "string",
+            "format", "uri"
+        }
+        "entities": {
+            "type": "object"
```

Comment:
Changed entities from a string type to an object type. Meaning "since the type validation keyword is object we can use the properties validation keyword to define a nested data structure." In this case for the entities we have a nested data structure of a dict. Without knowing concrete key names like `foo` or `bar` I'm not sure we can go further into the nested dict.

```
entities:
     foo: 1
     bar: “/some/path”
```

Is it sufficient to check here and confirm that entities does exist and is a dict? If we want to restrict the types that the values can be, how can we iterate over varying keys? I considered the naive approach would be to iterate over the keys and return them. Then use this information to make the schema accurate and restrict their types. Essentially filling in the missing information. After reading the problems with [this approach](https://github.com/json-schema-org/json-schema-spec/issues/1378) I'm not sure that's the way we want to go

Reply:
I think it's fine for this work to leave it with just being a dictionary.